values/keycloak/keycloak.gotmpl

```diff
@@ -87,6 +88,39 @@ extraVolumes: |
   - name: theme
     emptyDir: {}
 
+{{- if $v.databases.keycloak.useOtomiDB }}
+extraEnv: |
+  - name: DB_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: "keycloak-db-app"
+        key: password
+  - name: DB_USER
+    valueFrom:
+      secretKeyRef:
+        name: "keycloak-db-app"
+        key: username
+  - name: DB_VENDOR
+    value: postgres
+  - name: DB_ADDR
+    value: "keycloak-db-rw.keycloak.svc.cluster.local"
+  - name: DB_PORT
+    value: "5432"
+  - name: DB_DATABASE
+    value: keycloak
+{{- else }}
+extraEnv: |
+  - name: KEYCLOAK_USER
+    value: {{ $k.adminUsername }}
+  - name: KEYCLOAK_PASSWORD
+    value: {{ $k.adminPassword }}
+{{- end }}
+
+{{- if $v.databases.keycloak.useOtomiDB }}
```

Comment:
This could be in the same if statement above, but this is better for readability I think.